integration-tests/sanity-functional-protractor/page_objects/api-po.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const AbstractPage = require('./abstract-po.js');
+const qs = require('querystring')
+const chakram = require('chakram');
+const port = 8181;
+const getMetadata = '/api/metadata';
+const getArtifact = '/api/artifact';
+
+function RestAPI() {
+
+    this.getMetadata = (queryParams) => {
+        const serviceUrl = `${AbstractPage.getDomain()}:${port}${getMetadata}`;
```

Comment:
Combining here old fashioned JS classes with ES6 string literals so why no just declare class here.

Reply:
I do not get your point, will talk f2f ;)